tests/test_database_access.py

```diff
@@ -0,0 +1,189 @@
+from unittest.mock import patch
+import pytest
+from api.database_access.control import DBControl
+from api.database_access.commands import  GetLightCurve, GetDetections, GetNonDetections, BaseCommand
+from api.database_access.commands import InterfaceNotFound
+from api.database_access.interfaces import DBInterface, PSQLInterface, MongoInterface
+from api.database_access.interfaces import ObjectNotFound
+
+
+ZTF_ID = "ztf"
+ATLAS_ID = "atlas"
+
+def test_get_light_curve_mongo(psql_service, client):
+  command = GetLightCurve("ATLAS1", ATLAS_ID)
+  result = command.execute()
+
+  assert len(result["detections"]) == 1
+  assert result["detections"][0]["oid"] == "ATLAS1"
+
+  assert len(result["non_detections"]) == 1
+  assert result["non_detections"][0]["oid"] == "ATLAS1"
```

Comment:
I thought this would not work, because the fixture for the mongo service is not used here. Maybe it's just a coincidence that the mongo docker container is running at the time the tests start.

Reply:
As far I understand the test, none uses directly the fiixture service.

Reply:
Correct, there's no test that actually uses de service (ip + port I think is the return of the fixture) but it performs the check. I'm not sure if the check is performed anyway. I believe not, so it was just a matter of luck wether the service is ready to use or not.